app/persistence/fabric/CRUDService.ts

```diff
@@ -82,31 +82,51 @@ export class CRUDService {
 	 * @param {*} txid
 	 * @param {*} from
 	 * @param {*} to
+	 * @param {*} page
+	 * @param {*} size
 	 * @param {*} orgs
 	 * @returns
 	 * @memberof CRUDService
 	 */
-	getTxList(
+	async getTxList(
 		network_name: any,
 		channel_genesis_hash: any,
 		blockNum: any,
 		txid: any,
 		from: any,
 		to: any,
-		orgs: string
+		orgs: string,
+		page: any,
+		size: any
 	) {
+		var countOfTxns: any;
```

Comment:
[nitpick]: type will be number.

Reply:
The sqlQuery is expected to return the count of transactions and incase if it fails to do so, it returns "null".  The second thing is if page=1 then we would be sending the "noOfPages" in response and for the remaining pages we would send it as "null". So considering these cases assigned the type as "any" to countOfTxns.

Reply:
pipes can be used something like this `let countOfTxns : number | Error`

Reply:
I have tried with these combinations: var countOfTxns: number | null = null; Error is not a type and can't be defined. since we are applying arithmetic operation on countOfTransactions to derive noOfpages, the allowed types are number, any, bigInt, enum. I cant define type to be only "number" since there is a case of countOfTransactions to be null.
Below are the errors with wrong type definition:
 error TS2322: Type 'unknown' is not assignable to type 'number'.
 error Type 'unknown' is not assignable to type 'number | Error'.
 error The left-hand side of an arithmetic operation must be of type 'any', 'number', 'bigint' or an enum type.
So, I can define type to be "any" (or) number | unknown.